core/org.osate.xtext.aadl2.ui/src/org/osate/xtext/aadl2/ui/labeling/Aadl2LabelProvider.java

```diff
@@ -391,6 +393,30 @@ String text(Feature ele) {
 		return "Feature " + ele.getName();
 	}
 
+	String text(FlowImplementation flowimpl) {
+		String ret;
+
+		ret = "Flow ";
```

Comment:
Remove space character from this string.

Reply:
deleted extra space